obspy/io/sac/sactrace.py

```diff
@@ -66,6 +66,18 @@
     # write a binary SAC file for a Sun machine
     sac.write(filename, byteorder='big')
 
+Build a SACTrace from a header dictionary and data array
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    header = {'kstnm': 'ANMO', 'kcmpnm': 'BHZ', stla=40.5, stlo=-108.23,
+              'evla': -15.123, 'evlo': 123, 'evdp': 50, 'nzyear': 2012,
+              'nzjday': 123, 'nzhour': 13, 'nzmin': 43, 'nzsec': 17,
+              'nzmsec': 100, delta=1.0/40}
```

Comment:
> delta is not correctly set here.

Thanks for those catches @QuLogic.

Good example why it's better to put self contained examples as doctests in those docstrings.. e.g.:
```
    >>> my_array = np.ones(10)
    >>> header = {
    ...     'kstnm': 'ANMO', 'kcmpnm': 'BHZ', 'stla': 40.5, 'stlo': -108.23,
    ...     'evla': -15.123, 'evlo': 123, 'evdp': 50, 'nzyear': 2012,
    ...     'nzjday': 123, 'nzhour': 13, 'nzmin': 43, 'nzsec': 17,
    ...     'nzmsec': 100, 'delta': 1.0/40}
    >>> sac_trace = SACTrace(data=my_array, **header)
```
instead of
```
.. code:: python
    header = {'kstnm': 'ANMO', 'kcmpnm': 'BHZ', 'stla': 40.5, 'stlo': -108.23,
              'evla': -15.123, 'evlo': 123, 'evdp': 50, 'nzyear': 2012,
              'nzjday': 123, 'nzhour': 13, 'nzmin': 43, 'nzsec': 17,
              'nzmsec': 100, 'delta': 1.0/40}
    sac = SACTrace(data=my_array, **header)
```